.moai/specs/SPEC-V3R3-BRAIN-001/spec-compact.md

```
@@ -0,0 +1,125 @@
+**created_at**: 2026-05-04 | **updated_at**: 2026-05-04
+**Harness**: standard
+
+---
+
+## Goal (1 sentence)
+
+Introduce a pre-spec ideation workflow `/moai brain` that converts vague user ideas into validated product proposals with SPEC decomposition candidates AND a paste-ready Claude Design handoff package.
+
+## Workflow Position
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)